scenarioo-client/app/app.ts

```diff
@@ -17,7 +17,7 @@
 
 
 import {addRoutes} from "./app.routes";
-import * as angular from "angular";
+declare var angular: angular.IAngularStatic;
```

Comment:
Otherwise you get the warning that angular is imported multiple times (because you are importing angular multiple times). 

This way you just declare that there is a global Angular definition.

Reply:
Ok strange, I though webpack would only import it once nevertheless.